README.md

```diff
@@ -1,75 +1,88 @@
-| index | [example]     |
-|-------|---------------|
+| [README.md](#) | [Add-on template](https://github.com/mozilla/shield-studies-addon-template/) | [Engineering hints](#engineering-and-process) | [More documentation](./docs/) | [Shield - Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield) |
+| -------------- | ---------------------------------------------------------------------------- | --------------------------------------------- | ----------------------------- | ---------------------------------------------------------------- |
 
 
-# Shield Studies Addon Utils [![Build Status](https://travis-ci.org/tombell/travis-ci-status.svg?branch=master)](https://travis-ci.org/tombell/travis-ci-status)
+# Shield Studies Add-on Utils
 
-- Instrument your Firefox Addon!
-- Build Shield Study (Normandy) compatible addons without having to think very much.
+[![Build Status](https://travis-ci.org/mozilla/shield-studies-addon-utils.svg?branch=master)](https://travis-ci.org/mozilla/shield-studies-addon-utils)
 
-## Assumptions
+A Firefox JavaScript module to be bundled with shield study add-ons (as `StudyUtils.jsm`). Provides these capabilities:
 
-- You are building a [LEGACY ADDONS](https://developer.mozilla.org/en-US/Add-ons/Legacy_add_ons).  To deploy these after 57, you will need the magic special signing.
--  Jetpack / addon-sdk is NOT SUPPORTED.
-- WebExtensions are not strong enough
+1.  **Suggest variation for a client**
+2.  **Report study lifecycle data** using Telemetry
+3.  **Report feature interaction and success data** using Telemetry
+4.  **Annotate Telemetry Environment** to mark the user as special in the `main` ping.
```

Comment:
Thanks, I added to the list. Regarding #107: I am making a new PR with more dramatic changes to this repo, it is based upon your #107, so feel free to merge #107 right away (much better with jsdoc than without!)

Reply:
The reason why it isn't merged is because it would require us to publish a new package in npm. I do not have approval from gregglind to do that (nor have I ever done it before). @gregglind , would it be okay to merge #107 ? Would we want to re-publish?

Reply:
You are right, better not to merge #107 right now, better to have it baked into the next release, which will be v5. A first PR coming next week! :)